Blockzilla/BlockerEnabledDetector.swift

```diff
@@ -55,7 +55,7 @@ private class BlockerEnabledDetector9: BlockerEnabledDetector, SFSafariViewContr
         enabled = true
         self.callback = callback
 
-        let detectURL = URL(string: "http://localhost:\(server.port)/enabled-detector")!
+        let detectURL = URL(string: "http://localhost:\(server?.port)/enabled-detector")!
```

Comment:
I think this will print `http://localhost:Optional(<port number>)/enabled-detector`. We could either unwrap or map it.